src/hw-monitor.cpp

```diff
@@ -6,6 +6,8 @@
 #include <limits>
 #include <sstream>
 
+#include "ds/ds6/ds6-private.h"
```

Comment:
Here we are in the old `hw_monitor.cpp` file
Maybe it's better to split the new class to a new file if it needs to include `ds6` and place it in the `ds6` folder.

Reply:
done